### PR TITLE
Fix ob doc

### PR DESCRIPTION
**Pull request by tomsmalley**
I have:

  - [x] Based work on latest `develop` branch
  - [x] Looked for lint in my changes with `hlint .` (lint found code you did not write can be left alone)
  - [x] Run the test suite: `$(nix-build -A selftest --no-out-link)`
  - [ ] (Optional) Run CI tests locally: `nix-build release.nix -A build.x86_64-linux --no-out-link` (or `x86_64-darwin` on macOS)

### CODE REVIEW

File: lib/selftest/src/Obelisk/SelfTest.hs [hunk 1]
```diff
@@ -243,6 +243,8 @@ main' isVerbose httpManager obeliskRepoReadOnly = withInitCache $ \initCache ->
       uu <- update
       assertRevEQ u uu
 
+    it "can run 'ob doc'" $ inTmpObInit $ \_ -> runOb ["doc", "reflex"]
```
Comment (3noch) [inline lib/selftest/src/Obelisk/SelfTest.hs:246]: Does this actually fail without the fix?

Comment (tomsmalley) [reply lib/selftest/src/Obelisk/SelfTest.hs:246]: Yep

Comment (tomsmalley) [reply lib/selftest/src/Obelisk/SelfTest.hs:246]:
```
  src/Obelisk/SelfTest.hs:246:5: 
  1) blank initialized project can run 'ob doc'
       uncaught exception: ReThrownException
       
       Ran commands: 
       cd /tmp/test λ-61f9295e548d4478
       /nix/store/3kqc2wmvf1jkqb2jmcm7rvd9lf4345ra-coreutils-8.31/bin/cp -rT '/tmp/init Cache λ-697a7e209a58e822' '/tmp/test λ-61f9295e548d4478'
       which /nix/store/3kqc2wmvf1jkqb2jmcm7rvd9lf4345ra-coreutils-8.31/bin/cp
       ob doc reflex
       which ob
       cd /nix/store/91k5g8wvnb4bk75m9nj3rqpdvzw60ix3-impl
       
       Exception: error running: ob doc reflex
       exit status: 2
       stderr: Alert: No valid, buildable packages found
```

**Comment by 3noch** [comment]
This passed CI.

**Pull request merged**